messaging/src/main/java/org/axonframework/eventhandling/GapAwareTrackingToken.java

```diff
@@ -43,6 +43,8 @@
  */
 public class GapAwareTrackingToken implements TrackingToken, Serializable {
 
+    private static final long serialVersionUID = -3190388158060110593L;
```

Comment:
I don't think we should do this since there are probably projects using 4.0 that have `GapAwareTrackingToken` already serialized. Deserialization will fail since we provided a new `serialVersionUID`.

Reply:
So, I understand your concern, but as we were missing a `serialVersionUID`, the JVM would automatically create on for us if it would be serialized. 
It does so through the following operation: 
```
ObjectStreamClass.lookup({class-to-create-serial-version-uid-for}).getSerialVersionUID();
```
The number you see there is what _that_ function creates.
The number I put in the legacy version of both tokens, is generated through that exact same function I just shared, but then on branch `axon-3.4.x`, hence what old users would've had.

As such, this deviation should be completely fine.